docker/prod.yml

```diff
@@ -1,5 +0,0 @@
-version: "3.9"
```

Comment:
This file is going to be added back once we're sure it works well on dev, right?

Reply:
no, we don't need it anymore